aidants_connect_habilitation/forms.py

```diff
@@ -0,0 +1,116 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.forms import formset_factory, ChoiceField, CharField, BaseFormSet
+from phonenumber_field.formfields import PhoneNumberField
+from phonenumber_field.widgets import PhoneNumberInternationalFallbackWidget
+
+from aidants_connect.constants import RequestOriginConstants
+from aidants_connect.forms import PatchedErrorListForm, PatchedErrorList
+from aidants_connect_habilitation import models
+from aidants_connect_habilitation.models import AidantRequest
+from aidants_connect_web.models import OrganisationType
+
+
+class IssuerForm(PatchedErrorListForm):
+    phone = PhoneNumberField(
+        initial="",
+        region=settings.PHONENUMBER_DEFAULT_REGION,
+        widget=PhoneNumberInternationalFallbackWidget(
+            region=settings.PHONENUMBER_DEFAULT_REGION
+        ),
+        required=False,
+    )
+
+    class Meta:
+        model = models.Issuer
+        exclude = ["draft_id"]
+
+
+class OrganisationRequestForm(PatchedErrorListForm):
+    type = ChoiceField(required=True, choices=RequestOriginConstants.choices)
+
+    zipcode = CharField(
+        label="Code Postal",
+        max_length=10,
+        error_messages={
+            "required": "Le champ 'code postal' est requis",
```

Comment:
Ui bonjour c'est la collègue pénible qui veut de la bonne typo et des points à la fin des phrases…

```suggestion
            "required": "Le champ « code postal » est obligatoire.",
```